.github/workflows/create_mlflow.yml

```diff
@@ -48,6 +48,11 @@ jobs:
       - name: Run bandit
         run: bandit -r ../exploding_springfield -c ../exploding_springfield/.bandit.yml
 
+      - name: Reclaim space
```

Comment:
Maybe we can reclaim the space before any other steps? Think it would separate this more clearly.

Same for all other WFs.

Reply:
Y was a workingw title, will fix